Dockerfile

```diff
@@ -1,29 +1,30 @@
 FROM node:20-alpine AS builder
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
 
 WORKDIR /app
-COPY package.json .
-COPY yarn.lock .
 
-RUN yarn install --network-timeout 600000
-
-COPY src/ src
-COPY nginx.conf .
-COPY tsconfig.json .
-COPY vite.config.mts .
 COPY . .
 
 RUN echo `git rev-parse --short HEAD` > health.html
-RUN yarn build
+
+RUN if [ -d "./devtron-fe-lib" ]; then rm -rf packages/devtron-fe-lib; cp -r ./devtron-fe-lib packages/devtron-fe-lib; fi
+
+RUN pnpm i
+RUN pnpm run build:prod
```

Comment:
add command for linting as well or CI/CD for FE lib

Reply:
updated the CI workflow in both dashboard and devtron-fe-lib